R/BenchmarkResult.R

```diff
@@ -359,9 +366,7 @@ BenchmarkResult = R6Class("BenchmarkResult",
       }
 
       ResampleResult$new(self$data, view = needle)
-    }
-  ),
-
+    }),
```

Comment:
@mllg 

IIRC we discussed about this pattern/change the last time already and agreed to allow both options?

Reply:
Tested this on another repo and in our old mlr-style this was not changed by {styler} -> @lorenzwalthert can we remove this styling? :)

Reply:
Indeed, there was a transformer for that,but for some reason, it was commented out in the revision of your fork I used when c/p the transformers. Adressed in https://github.com/mlr-org/styler.mlr/pull/7.

Reply:
Is this a possible misunderstanding?

I meant to say that we want to allow the 

```
    }
  )
``` 

pattern. And avoid a styling into `})`.
I think this is why I commented out this transformer at some point.

I see that in the pull you explicitly commented it in again.
Assuming that it was commented out in this current draft - why did this styling happen then? 🤔 
Or am I on the wrong path?

Reply:
Yup, there was a misunderstanding (and also merge conflicts). We want to activate the rule that adds a line break if there is none and remove the rule that removes the line break. Force pushed to https://github.com/mlr-org/styler.mlr after rebasing on main.